doc/lightning_tensor/installation.rst

```diff
@@ -23,8 +23,7 @@ Lightning-Qubit needs to be 'installed' by ``pip`` before Lightning-Tensor (comp
 
     git clone https://github.com/PennyLaneAI/pennylane-lightning.git
     cd pennylane-lightning
-    pip install -r requirements.txt
-    pip install cutensornet-cu12
+    pip install -r requirements-dev.txt
```

Comment:
The requirements-dev is for developers. We should be explicit about the required packages to be installed here instead of installing all the dev deps.